types/request/index.d.ts

```diff
@@ -167,6 +168,23 @@ declare namespace request {
         useQuerystring?: boolean;
     }
 
+    interface AgentOptions {
```

Comment:
This seems to be copy/paste from `SecureContextOptions` from the node typings?

Reply:
@SimonSchick I wrote it based on node docs so it may be similar or even exactly the same. At the moment just as `any` which is definitely bad

Reply:
That's ok, lets try to reduce the references to the node module tho :)

Reply:
You mean to import Node typings and just assign them here?

Reply:
Done.